ThunderTable/TableViewController.swift

```diff
@@ -153,11 +153,11 @@ open class TableViewController: UITableViewController {
 		NotificationCenter.default.removeObserver(dynamicChangeObserver)
 	}
     
-    public var inputDictionary: [String: Any?]? {
+    public var inputDictionary: [String: Any]? {
```

Comment:
If you are creating an empty dictionary below, is there any point in this being optional? You are always going to return at least an empty dictionary which is a non-nil object?